static/js/biomedical/disease/drug_table.tsx

```diff
@@ -1,37 +1,48 @@
 import React from "react";
 
 import { GRAPH_BROWSER_REDIRECT } from "../bio_charts_utils";
-import { CompoundDiseaseTreatmentData } from "./chart";
+import {
+  CompoundDiseaseContraindicationData,
+  CompoundDiseaseTreatmentData,
+} from "./chart";
 
+export interface DrugTableColumnType {
+  id: string;
+  name: string;
+}
 export interface DrugTableProps {
-  data: CompoundDiseaseTreatmentData[];
+  columns: DrugTableColumnType[];
+  data: Record<never, never>[];
```

Comment:
Instead of writing an additional function which converts `CompoundDiseaseContraindicationData` and `CompoundDiseaseTreatmentData`, I created another generic interface called `DrugDataType`, which removes the ambiguity with the use of index signature syntax.